### PR TITLE
test: add parseSysctl and parseNetwork unit test

### PR DESCRIPTION
**Pull request by ZouRui89**
Signed-off-by: Zou Rui <21751189@zju.edu.cn>

### Ⅰ. Describe what this PR did
 add parseSysctl and parseNetwork unit test

### Ⅱ. Does this pull request fix one issue?

NONE

### CODE REVIEW

File: cli/container_test.go [hunk 1]
```diff
@@ -329,3 +329,97 @@ func Test_parseDeviceMappings(t *testing.T) {
 		})
```
Comment (ZouRui89) [inline cli/container_test.go]: @allencloud As you can see, in the unit test case generated by vscode, we can only detect the existence of an error, true or false.

File: cli/container_test.go [hunk 2]
```diff
@@ -329,3 +329,97 @@ func Test_parseDeviceMappings(t *testing.T) {
...
+			},
+		},
+	}
+
+	for _, testCase := range testCases {
+		name, ip, error := parseNetwork(testCase.input)
+		assert.Equal(t, testCase.expect.err, error)
+		assert.Equal(t, testCase.expect.network.name, name)
+		assert.Equal(t, testCase.expect.network.ip, ip)
+	}
```
Comment (ZouRui89) [inline cli/container_test.go]: @allencloud However, we need to know whether or not the type of the error is the exact one we are expecting. In this case, I don't think the unit test framework generated by vscode is good enough.

**Comment by Letty5411** [comment]
LGTM

File: cli/container_test.go (hunk 1 above)
Comment (allencloud) [reply cli/container_test.go]: yeah, I think it has some drawbacks.

**Pull request merged**